packages/models/src/lib/report.ts

```diff
@@ -39,6 +39,7 @@ export const reportSchema = packageVersionSchema({
   .merge(
     z.object(
       {
+        commit: z.string().optional(),
```

Comment:
I added the `commit` to the model. Seem's we missed it so far as it is used only in the `md` report and `stdout` directly